### PR TITLE
Install errors and messages

### PR DESCRIPTION
**Pull request by cbridgha**
This PR has changes reacting to results coming from the installServer Action possibly including versionless / platforms in the server.xml.

2 new Messages have been added explaining issues with bad platform names and the possibility no platform could be determined with the collection of passed features.

The existing messages in place take care of other scenarios like feature not found, resoltion problems etc...  so no change there.

2 new tests were also added that look for the new messages, The fat tests in general were restructured to use one of two repo's, the later is a new copy containing versionless features - Versionless tests are using the new local repo.

### CODE REVIEW

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

File: dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops [hunk 1]
```diff
@@ -949,3 +949,11 @@ ERROR_INAUTHENTIC_PUBLIC_KEY=CWWKF1514E: The {0} public key ID does not match th
 ERROR_INAUTHENTIC_PUBLIC_KEY.explanation=The public key might have changed. 
 ERROR_INAUTHENTIC_PUBLIC_KEY.useraction=If the public key is a local file, make sure to provide the correct file.
 
+ERROR_MISSING_PLATFORM_NAME=CWWKF1515E: The platform {0} couldn't be found, no versionless features will be installed.
```
Comment (dmuelle) [inline dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops]:
```suggestion
ERROR_MISSING_PLATFORM_NAME=CWWKF1515E: The {0} platform couldn't be found. No versionless features can be installed.
```

File: dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops [hunk 2]
```diff
@@ -949,3 +949,11 @@ ERROR_INAUTHENTIC_PUBLIC_KEY=CWWKF1514E: The {0} public key ID does not match th
...
+ERROR_MISSING_PLATFORM_NAME.explanation=The platform name passed by the config or Env Var wasn't found.
```
Comment (dmuelle) [inline dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops]:
```suggestion
ERROR_MISSING_PLATFORM_NAME.explanation=The platform name that is specified by the configuration or environment variable wasn't found.
```

File: dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops [hunk 3]
```diff
@@ -949,3 +949,11 @@ ERROR_INAUTHENTIC_PUBLIC_KEY=CWWKF1514E: The {0} public key ID does not match th
...
+ERROR_MISSING_PLATFORM_NAME.useraction=Check the spelling of the platform name-version.
```
Comment (dmuelle) [inline dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops]:
```suggestion
ERROR_MISSING_PLATFORM_NAME.useraction=Check the spelling of the platform name and version, for example, microProfile-6.1.
```

File: dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops [hunk 4]
```diff
@@ -949,3 +949,11 @@ ERROR_INAUTHENTIC_PUBLIC_KEY=CWWKF1514E: The {0} public key ID does not match th
...
+
+ERROR_NO_DERIVED_PLATFORM=CWWKF1516E: Platform couldn't be determined, the following versionless features won't be installed: {0}.
```
Comment (dmuelle) [inline dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops]:
```suggestion
ERROR_NO_DERIVED_PLATFORM=CWWKF1516E: The platform couldn't be determined. The following versionless features can't be installed: {0}.
```

File: dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops [hunk 5]
```diff
@@ -949,3 +949,11 @@ ERROR_INAUTHENTIC_PUBLIC_KEY=CWWKF1514E: The {0} public key ID does not match th
...
+ERROR_NO_DERIVED_PLATFORM.explanation=Platform was not set, or couldn't be derived from existing versioned features.
```
Comment (dmuelle) [inline dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops]:
```suggestion
ERROR_NO_DERIVED_PLATFORM.explanation= The platform was not set or couldn't be derived from existing versioned features. 
```

File: dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops [hunk 6]
```diff
@@ -949,3 +949,11 @@ ERROR_INAUTHENTIC_PUBLIC_KEY=CWWKF1514E: The {0} public key ID does not match th
 ERROR_INAUTHENTIC_PUBLIC_KEY.explanation=The public key might have changed. 
 ERROR_INAUTHENTIC_PUBLIC_KEY.useraction=If the public key is a local file, make sure to provide the correct file.
 
+ERROR_MISSING_PLATFORM_NAME=CWWKF1515E: The platform {0} couldn't be found, no versionless features will be installed.
+ERROR_MISSING_PLATFORM_NAME.explanation=The platform name passed by the config or Env Var wasn't found.
+ERROR_MISSING_PLATFORM_NAME.useraction=Check the spelling of the platform name-version.
+
+ERROR_NO_DERIVED_PLATFORM=CWWKF1516E: Platform couldn't be determined, the following versionless features won't be installed: {0}.
+ERROR_NO_DERIVED_PLATFORM.explanation=Platform was not set, or couldn't be derived from existing versioned features. 
+ERROR_NO_DERIVED_PLATFORM.useraction=Add platform for versionless features
```
Comment (dmuelle) [inline dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops]:
```suggestion
ERROR_NO_DERIVED_PLATFORM.useraction=Add a platform for versionless features.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review

**Comment by cbridgha** [comment]
#build

**Review by dmuelle — DISMISSED** [review]
LGTM, thanks

**Comment by cbridgha** [comment]
Latest build with all changes: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_7eea0ES2Ee-ngPvsOTExmw

**Review by jjiwooLim — APPROVED** [review]
lgtm!

**Comment by cbridgha** [comment]
#libby

**Pull request merged**